src/ui.rs

```diff
@@ -122,64 +108,61 @@ pub fn show_card(score_card: &scorecard::ScoreCardData) {
 
 pub fn show_hand(hand: &Dice) {
     fn print_color(s: &str, face: DieFace) {
-        match face {
-            1 => print!("{}", Red.bg(White).paint(&s)),
-            2 => print!("{}", Magenta.bg(White).paint(&s)),
-            3 => print!("{}", BrightGreen.bg(White).paint(&s)),
-            4 => print!("{}", BrightCyan.bg(White).paint(&s)),
-            5 => print!("{}", Green.bg(White).paint(&s)),
-            6 => print!("{}", Black.bg(White).paint(&s)),
-            _ => print!("{}", Black.bg(White).paint(&s)),
-        }
+        print!(
+            "{}",
+            match face {
+                1 => Red.bg(White).paint(&s),
+                2 => Magenta.bg(White).paint(&s),
+                3 => BrightGreen.bg(White).paint(&s),
+                4 => BrightCyan.bg(White).paint(&s),
+                5 => Green.bg(White).paint(&s),
+                6 => Black.bg(White).paint(&s),
+                _ => Black.bg(White).paint(&s),
+            }
+        );
     }
 
-    const SIX: &[&str] = &[
-        "●   ●", //
-        "●   ●", //
-        "●   ●", //
-    ];
-    const FIVE: &[&str] = &[
-        "●   ●", //
-        "  ●  ",   //
-        "●   ●", //
-    ];
-    const FOUR: &[&str] = &[
-        "●   ●", //
-        "     ",     //
-        "●   ●", //
-    ];
-    const THREE: &[&str] = &[
-        "    ●", //
-        "  ●  ", //
-        "●    ", //
-    ];
-    const TWO: &[&str] = &[
-        "●    ", //
-        "     ",   //
-        "    ●", //
-    ];
-    const ONE: &[&str] = &[
-        "     ",   //
-        "  ●  ", //
-        "     ",   //
+    #[rustfmt::skip]
```

Comment:
I actually didn't know about this attribute, but I figured there had to be a better way to get rustfmt to ignore something so I googled it. I had tried to find this before but I think something recently stabilized made it possible.
I rearranged this into indexing into an array, since it seems cleaner to me.